package.json

```diff
@@ -66,17 +66,17 @@
     "docker:compose:service:up": "docker-compose -p hrm -f hrm-service/docker-compose.yml up -d",
     "lint": "eslint --ext ts .",
     "lint:fix": "npm run lint -- --fix",
-    "localstack:bootstrap": "cdklocal bootstrap",
-    "localstack:deploy": "cdklocal deploy --all --outputs-file ./cdk/outputs.json",
-    "localstack:destroy": "cdklocal destroy --all",
-    "localstack:diff": "cdklocal diff",
+    "localstack:bootstrap": "cross-env AWS_REGION=eu-west-1 cdklocal bootstrap",
```

Comment:
AWS_REGION set as a local env var was causing localstack to use different regions for different calls/dependent tests. Hardcoding overrides whatever the user may have set in their local env and standardizes localstack to `eu-west-1`